packages/f2elint/src/config/_vscode/settings.json.ejs

```diff
@@ -1,12 +1,19 @@
 {
   "eslint.validate": ["javascript", "javascriptreact", "typescript", "typescriptreact", "vue"],
-  "stylelint.validate": ["css", "less", "sass", "scss"],
+  <%_ if (enableStylelint) { _%>
+  "stylelint.validate": [<%- stylelintExt.map((ext) => `"${ext.replace(/^\./, '')}"`).join(',') %>],
+  <%_ } _%>
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true,
-    "source.fixAll.stylelint": true,
+    "source.fixAll.eslint": true<%= enableStylelint || enableMarkdownlint ? ',' : ''%>
+    <%_ if (enableStylelint) { _%>
+    "source.fixAll.stylelint": true<%= enableMarkdownlint ? ',' : ''%>
+    <%_ } _%>
+    <%_ if (enableMarkdownlint) { _%>
     "source.fixAll.markdownlint": true
+    <%_ } _%>
   },
-  "editor.formatOnSave": false,
+  "editor.formatOnSave": <%= enablePrettier ? 'true,' : 'false' %>
```

Comment:
formatOnSave和codeActionsOnSave的执行顺序是咋样的，会不会打架呀

Reply:
https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

NOTE: If you are seeing conflicts between Prettier and ESLint this is because you don't have the right ESLint or TSLint rules set as explained in the Prettier documentation.

在我们有配置 `prettier/@typescript-eslint`，让 eslint 不再管格式问题，随不会出现打架的情况

Reply:
嗯，顺序感觉不是很重要，规则不冲突就行